public/index.css

```diff
@@ -0,0 +1,97 @@
+html, body {
+  height: 100%;
+}
+
+html {
+  display: table;
```

Comment:
wow this is crazy! It's amazing that this works. I'm impressed you got this solution to work. So while this meets the specs and you don't have to change it I recommend you find a solution that doesn't involve hacky css tricks on the `html` or `body` element. I mainly recommend this to better simulate a real-world situation where you would build a component as complex as this. Look into flexbox